test/info.py

```diff
@@ -173,6 +174,7 @@
     (   0x7404,     'lpc11u35_6lowpan_borderrouterethernet_if', None,               'mbed-6LoWPAN-Border-Router-ETHERNET'   ),
     (   0xC000,     'lpc11u35_cocorico_if',                     None,               'CoCo-ri-Co'                            ),
     (   0x5500,     'lpc11u35_gr_peach_if',                     None,               None                                    ), # TODO - Set to 'Renesas-GR-PEACH' once this target builds
+    (   0x5501,     'lpc11u35_gr_lychee_if',                    None,               None                                    ), # TODO - Set to 'Renesas-GR-LYCHEE' once this target builds
```

Comment:
Please add this board ID to the `BOARD_ID_SUPPORTING_PAGE_ERASE` list near the bottom of `info.py`, since you have enabled page erase on this board.